### PR TITLE
Add line to README that project is alpha 

### PR DESCRIPTION
**Pull request by alexchaomander**
### Motivation and Context
Per discussion with @timlaverty and @evchaki, we want to communicate to the community that the Semantic Kernel is very early and will rapidly evolve. Also want to say that we invite the community to build the project together. 

Purpose is to show that we do not know everything and that we're open to feedback.

### Description
Line at the top of the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 [![License: MIT](https://img.shields.io/github/license/microsoft/semantic-kernel)](https://github.com/microsoft/semantic-kernel/blob/main/LICENSE)
 [![dotnet](https://github.com/microsoft/semantic-kernel/actions/workflows/dotnet-ci.yml/badge.svg?branch=main)](https://github.com/microsoft/semantic-kernel/actions/workflows/dotnet-ci.yml)
 
+> ⚠️ **NOTE**: This project is in early alpha and just like AI will evolve quickly. We invite you to join us in developing the Semantic Kernel together!
```
Comment (shawncal) [inline README.md:7]:
```suggestion
> ⚠️ **NOTE**: This project is in early alpha and, just like AI, will evolve quickly. We invite you to join us in developing the Semantic Kernel together!
```

**Review by shawncal — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -4,6 +4,8 @@
 [![License: MIT](https://img.shields.io/github/license/microsoft/semantic-kernel)](https://github.com/microsoft/semantic-kernel/blob/main/LICENSE)
 [![dotnet](https://github.com/microsoft/semantic-kernel/actions/workflows/dotnet-ci.yml/badge.svg?branch=main)](https://github.com/microsoft/semantic-kernel/actions/workflows/dotnet-ci.yml)
 
+> ⚠️ **NOTE**: This project is in early alpha and, just like AI, will evolve quickly. We invite you to join us in developing the Semantic Kernel together!
```
Comment (timlaverty) [inline README.md:7]: Would it be worth adding an ask for them to, "Please contribute by using GitHub Discussions, opening GitHub Issues, sending us PRs, and joining our Discord Channel?"

**Review by timlaverty — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (dluc) [inline README.md:7]: * I would remove the danger symbol, replace it with an info icon like ℹ️ (https://emojipedia.org/information/)
* nit: pls wrap the long line, thanks

Comment (alexchaomander) [reply README.md]: done

**Review by dluc — APPROVED** [review]

**Review by shawncal — APPROVED** [review]

**Pull request merged**